components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.html

```diff
@@ -49,11 +49,18 @@ <h2 slot="title">Create {{ objectNoun | titlecase }}</h2>
             Also create owner, editor, and viewer policies for this project.
           </chef-checkbox>
         </div>
-        <div class="dropdown-margin" *ngIf="objectNoun != 'project'">
+        <div class="dropdown-margin" *ngIf="objectNoun === 'token'">
+          <app-resource-dropdown
```

Comment:
The new dropdown for policies; it only shows up on the tokens page. (Still need to add permissions guard around it in follow-up PR.)
It is really just this block of HTML and a couple lines of code in the code-behind.